src/networkmessage.h

```diff
@@ -146,7 +146,7 @@ class NetworkMessage
 		};
 
 		NetworkMessageInfo info;
-		uint8_t buffer[NETWORKMESSAGE_MAXSIZE];
+		std::array<uint8_t, NETWORKMESSAGE_MAXSIZE> buffer;
```

Comment:
This was the easiest way to make `NetworkMessage` copyable, and it's better anyway.

Reply:
Well I solved this like that:
https://github.com/otland/forgottenserver/pull/2978/files#diff-a4a151297bcd412ef2b753d4b0b66f0614e336a866ea57688569c46352f5ac8bR47-R49